application/src/main/java/org/togetherjava/tjbot/features/mediaonly/MediaOnlyChannelListener.java

```diff
@@ -45,7 +53,22 @@ public void onMessageReceived(MessageReceivedEvent event) {
         }
 
         if (messageHasNoMediaAttached(message)) {
-            message.delete().flatMap(any -> dmUser(message)).queue();
+            message.delete().flatMap(any -> dmUser(message)).queue(res -> {
+                logger.info("This is a Error handler of DM");
+            }, new ErrorHandler().handle(ErrorResponse.CANNOT_SEND_TO_USER, err -> {
+                logger.debug(err.toString());
+                logger.error(message.toString());
+                if (event.getAuthor().isBot()) {
+                    logger.info("Ready to handle Error is bot");
+                    return;
+                }
```

Comment:
it is already checked, u don't have to check it again

Reply:
Done removed in my local repo will push In the next commit